sbt-plugin-test/crossProjectRef/build.sbt

```diff
@@ -0,0 +1,8 @@
+lazy val crossProjectRef = crossProject.
+  crossType(CrossType.Pure).
+  in(file(".")).
+  settings(scalaVersion := "2.11.6") 
+
+lazy val crossProjectRefJS = crossProjectRef.js
+
+lazy val crossProjectRefJVM = crossProjectRef.jvm
```

Comment:
Could you name these differently, so it is clear that they are referenced (and not the reference itself)?